homeassistant/components/switchbee/light.py

```diff
@@ -0,0 +1,150 @@
+    )
+
+
+class SwitchBeeLightEntity(SwitchBeeDeviceEntity[SwitchBeeDimmer], LightEntity):
+    """Representation of an SwitchBee light."""
+
+    def __init__(
+        self,
+        device: SwitchBeeDimmer,
+        coordinator: SwitchBeeCoordinator,
+    ) -> None:
+        """Initialize the SwitchBee light."""
+        super().__init__(device, coordinator)
+        self._attr_is_on = False
+        self._attr_brightness = 0
+        self._attr_supported_color_modes = {ColorMode.BRIGHTNESS}
+        self._attr_available = True
+
+        self._update_attrs_from_coordinator()
+
+    @callback
+    def _handle_coordinator_update(self) -> None:
+        """Handle updated data from the coordinator."""
+        self._update_attrs_from_coordinator()
+        super()._handle_coordinator_update()
+
+    def _update_attrs_from_coordinator(self) -> None:
+
+        coordinator_device = cast(
+            SwitchBeeDimmer, self.coordinator.data[self._device.id]
+        )
+        brightness: int = coordinator_device.brightness
+
+        # module is offline
+        if brightness == -1:
+            # This specific call will refresh the state of the device in the CU
+            self.hass.async_create_task(self.async_refresh_state())
+
+            # if the device was online (now offline), log message and mark it as Unavailable
+            if self._is_online:
+                _LOGGER.warning(
+                    "%s light is not responding, check the status in the SwitchBee mobile app",
+                    self.name,
+                )
+                self._is_online = False
+
+            return
+
+        # check if the device was offline (now online) and bring it back
+        if not self._is_online:
+            _LOGGER.info(
+                "%s light is now responding",
+                self.name,
+            )
+            self._is_online = True
+
+        self._attr_is_on = bool(brightness != 0)
+
+        # 1-99 is the only valid SwitchBee brightness range
+        if 0 < brightness < 100:
+            self._attr_brightness = _switchbee_brightness_to_hass(brightness)
+
+    async def async_turn_on(self, **kwargs: Any) -> None:
+        """Async function to set on to light."""
+        if ATTR_BRIGHTNESS in kwargs:
+            state: int | str = _hass_brightness_to_switchbee(kwargs[ATTR_BRIGHTNESS])
+        else:
+            state = ApiStateCommand.ON
+            if self.brightness:
+                state = _hass_brightness_to_switchbee(self.brightness)
+
+        try:
+            await self.coordinator.api.set_state(self._device.id, state)
+        except (SwitchBeeError, SwitchBeeDeviceOfflineError) as exp:
+            raise HomeAssistantError(
+                f"Failed to set {self._attr_name} state {state}, {str(exp)}"
+            ) from exp
+
+        else:
+            # update the coordinator data manually if already know the Central Unit brightness data for this light
+            if isinstance(state, int):
```

Comment:
You could invert the check here and return early. Then you can un-indent a second time:
```python
            if not isinstance(state, int):
                # comment
                return
```